### DISCUSSION TITLE
streampark能否运行在1.24版本之后的k8s上

### DISCUSSION THREAD

**Discussion by Xiazki** [apache/streampark#4136] [Q&A]
streampark 部署k8s application作业需要使用docker环境，1.24之后的版本移除了docker，使用containerd；我准备部署streampark在最新的k8s集群上，目前是否兼容？

**Comment by monrg** [comment]
Kubernetes 1.24 版本之后，并没有完全移除 Docker，而是移除了对 Docker 作为容器运行时（Container Runtime） 的直接支持。这一变化与 Kubernetes 的容器运行时接口（Container Runtime Interface，CRI）有关。Kubernetes 1.24 及之后版本均可以使用docker
